README.md

```diff
@@ -1,32 +1,352 @@
+labelin SCT was used to denoise images (`sct_image --denoise`). Some centers have image specificity that made the
+manual
+reviewing hazardous and I preferred to only take images where I had a good confident level on my labels.
+
+As a result, only 20 subjects from D0b were retained and combined with D1b to create a new dataset comprising 38
+subjects (D2). Within this dataset, two subjects were transferred from the training dataset to the test dataset (
+sub-008_ses-headUp, sub-brnoUhb01). The mean age is 26.2 years old, with a standard deviation of 5.2.
+
+A five-fold training of nnUNetV2 3d_fullres model (M2) has been conducted for 1000 epochs, dice scores
+were between 0.65 and 0.75. Notably, no post-processing techniques yielded an improvement in scores under these
+circumstances.
+Inference on the D2 dataset with the M2 model helped me to correct my label and improve the D2 ground truth quality.
+
+> Refer to issue [issue#8 part 2)](https://github.com/ivadomed/model-spinal-rootlets/issues/8).
+
+#### D3)
+
+A new labeling of the D2 dataset with spinal level-depending values has been conducted. As a result of uncertainty, five
+images were excluded. The resultant Dataset D3 comprises 33 images, including 31 for training and 2 for testing (same as
+D2). This dataset features a subject mean age of 26.5 y.o, standard deviation of 5.5 and incorporates spinal
+level-specific spinal nerve segmentation.
+
+A five-fold training of nnUNet 3d_fullres model has been conducted for 1000 epochs, dice scores were
+between 0.4 and 0.6. No post-processing techniques led to an increase in scores under these
+conditions. Upon reviewing the progress.png graph, a subsequent training was conducted with 2000 epochs. This decision
+was based on the observation that the plateau had not been reached within the first 1000 epochs. The second training
+yielded a dice score also ranging between 0.4 and 0.6. However, it exhibited more folds with scores
+exceeding 0.5 compared to the first training conducted with 1000 epochs.
+
+> Refer to [issue#8 part 3)](https://github.com/ivadomed/model-spinal-rootlets/issues/8).
+
+</details>
+
+### C) Reproduce
+
+In the next section, all the instructions to reproduce the label files used in the final dataset will be described.
+
+nnUNet is used to train model but the dataset format is not BIDS:
+
+- From BIDS to
+  nnUNet [convert_bids_to_nnUnetv2.py](https://github.com/ivadomed/utilities/blob/main/dataset_conversion/convert_bids_to_nnUnetv2.py)
+- From nnUNet to BIDS [???](????)
+- Extract all image from bids to nnUNet
+  inference [extract_bids_subject.py](https://github.com/ivadomed/model-spinal-rootlets/blob/main/dataset_creation/extract_bids_subject.py)
+- Merge nnUNet
+  dataset [concat_nnUnet_dataset.py](https://github.com/ivadomed/model-spinal-rootlets/blob/main/dataset_creation/concat_nnUnet_dataset.py)
+
+#How to explain FLSEYES labeling ?
```

Comment:
Good question! We should record a short video explaining the procedure next week. See other SCT videos [here](https://www.youtube.com/@sctspinalcordtoolbox4189/videos).